src/components/GagesBarChartAnimation.vue

```diff
@@ -2699,175 +2699,178 @@
         aria-hidden="true"
       >
         <g class="cls-1">
-          <g id="bottom" transform="translate(0, -120)">
-          <g id="end-water-background">
-            <path
-              class="cls-2"
-              d="M913.1,1999.5c-62.7-1.3-191-1.2-253.9,0-80.9,1.4-215.7,1.2-286.8,0-194.2-3.3-373.1,1.6-373.1,1.6v162.7s123.3.3,186.7,1.6,191.1,1.1,254,0c80.8-1.5,215.6-1.2,286.8,0,194.1,3.2,373.1-1.6,373.1-1.6V2001.1S976.6,2000.8,913.1,1999.5Z"
-            />
-          </g>
-          <g id="end-tiny-gages">
-            <g>
+          <g
+            id="bottom"
+            transform="translate(0, -120)"
+          >
+            <g id="end-water-background">
               <path
-                class="cls-3"
-                d="M63,2147.5a86.7,86.7,0,0,1,9.4-1.1l7.3-.5c4.9-.4,9.2-1.3,14.2-1.7"
+                class="cls-2"
+                d="M913.1,1999.5c-62.7-1.3-191-1.2-253.9,0-80.9,1.4-215.7,1.2-286.8,0-194.2-3.3-373.1,1.6-373.1,1.6v162.7s123.3.3,186.7,1.6,191.1,1.1,254,0c80.8-1.5,215.6-1.2,286.8,0,194.1,3.2,373.1-1.6,373.1-1.6V2001.1S976.6,2000.8,913.1,1999.5Z"
               />
+            </g>
+            <g id="end-tiny-gages">
+              <g>
+                <path
+                  class="cls-3"
+                  d="M63,2147.5a86.7,86.7,0,0,1,9.4-1.1l7.3-.5c4.9-.4,9.2-1.3,14.2-1.7"
+                />
+                <path
+                  class="cls-3"
+                  d="M134.8,2149.1a107.7,107.7,0,0,1,12.1-1.2c2.9-.1,6-.1,8.9-.4a76.5,76.5,0,0,0,8.7-1.1"
+                />
+                <path
+                  class="cls-3"
+                  d="M131.2,2147.8l3.6-.7a6,6,0,0,1,2-.3h2.1a19,19,0,0,0,5.1-.7"
+                />
+                <path
+                  class="cls-3"
+                  d="M92.3,2148.4c2.9-.5,6-.8,9.4-1.1l7.3-.4c4.9-.4,9.2-1.3,14.1-1.8"
+                />
+                <path
+                  class="cls-3"
+                  d="M81.3,2147.5a110.2,110.2,0,0,1,12.1-1.2c3-.1,6.1-.1,9-.3l8.7-1.1"
+                />
+              </g>
               <path
-                class="cls-3"
-                d="M134.8,2149.1a107.7,107.7,0,0,1,12.1-1.2c2.9-.1,6-.1,8.9-.4a76.5,76.5,0,0,0,8.7-1.1"
+                class="cls-4"
+                d="M125.7,2096.9Z"
               />
               <path
-                class="cls-3"
```

Comment:
or are all of these just re-ordering of the paths within the group? If so, why would linting do that - order matters (since things can appear on top or below one another based on svg order)?